src/components/ColumnProfile.svelte

```diff
@@ -156,6 +156,7 @@
                             {totalRows}
                             topK={summary.topK}
                         />
+                        <p>{summary.textfact.description}</p>
```

Comment:
Lets discuss how we want to style this -- this is good for now for debugging but we'll want to change before merging in